cypress/e2e/types/types.ts

```diff
@@ -34,11 +34,20 @@ export type CredentialsMavenData = {
     settingFile?: any;
 };
 
+export type CredentialsJiraData = {
+    type: string;
+    name?: string;
```

Comment:
Name, Type , Email and Token are not optional

Reply:
`?` here is NOT because that is optional parameter, I need it here cause I have some replaceable types, only fields that are common for ALL those types can be without it. That is why it is there. Can't remove it, otherwise all code will be red and will not work.